### DISCUSSION TITLE
posts/2024/02/add-a-grid-background-image-to-blog/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#41] [General]
# posts/2024/02/add-a-grid-background-image-to-blog/

https://guozheng.rbind.io/posts/2024/02/add-a-grid-background-image-to-blog/

**Comment by CyrusYip** [comment]
好酷的 CSS 技巧！现在背景是灰色，灰色的字就不容易看清，建议调深色一些。例子 ：

![image](https://github.com/residualsun1/Residualsun/assets/60951091/9b602657-8ff6-4413-b626-d6fbd605b21b)

![image](https://github.com/residualsun1/Residualsun/assets/60951091/756a9dc4-62f6-48d2-b39a-2ddf141f6548)

**Reply by residualsun1** [reply]
回头看看怎么搭配调色。这两天还在观察这个背景和思源宋体搭不搭，对可阅读性影响咋样，说不准过一阵子还会恢复白色背景😄。

**Reply by residualsun1** [reply]
今天在阮一峰老师的周报里看到一个也是关于用CSS实现网格背景效果的[文章](https://spacexcode.com/blog/pure-css-grid-line/)，感觉讲得更详细全面。
